tox.ini

```diff
@@ -47,10 +63,23 @@ commands=
     python setup.py develop
     py.test -k test_verify_production_cot --random-order-bucket=none
 
+[testenv:check]
```

Comment:
We may want to name this something else, like lint, to avoid confusion with the github checks task that we'll add in taskgraph.

Reply:
Sorry, I'm not sure I follow. Are you talking about the new Github Checks (like https://github.com/mozilla-releng/scriptworker/pull/413/checks), or something else? Either way, renaming is not a big deal.

Reply:
Yes, aiui tc-github uses Github Checks for status reporting. When I saw this task running I thought it was the status update task. I don't think the rename is blocking, just pointing out potential confusion.